src/server/index.ts

```diff
@@ -0,0 +1,57 @@
+import { AnyChannel, Client, version } from 'discord.js';
+import dotenv from 'dotenv';
+import { _help } from '../run/command/help';
+import { _quote } from '../run/_quote';
+
+dotenv.config();
+const token = getEnv('DISCORD_TOKEN');
+const status = getEnv('STATUS_CHANNEL_ID');
+function getEnv(name: string): string {
+  const value = process.env[name];
+  if (!value) return '';
+  return value;
+}
+
+const client = new Client({
+  intents: ['GUILDS', 'GUILD_MESSAGES']
+});
+
+/* 接続時にクライアントの情報を提供する */
+function readyLog(client: Client): void {
+  const connectionClient = client.user;
+  const projectVersion = process.env.npm_package_version;
+  if (connectionClient == null) return;
+  console.info('============');
+  console.info('');
+  console.info('起動完了しました。');
+  console.info('');
+  console.info('接続クライアント> ' + connectionClient.username);
+  console.info('接続クライアントID> ' + connectionClient.id);
+  //
+  console.info(`接続クライアントバージョン> ${String(projectVersion)}`);
+  console.info('');
+  console.info('discord.js バージョン> ' + version);
+  console.info('');
+  console.info('============');
+}
+
+client.login(token).catch(console.error);
```

Comment:
```suggestion
client.login(token);
```
ログインの失敗をコンソール出力だけして無視するのは危ないかと